### PR TITLE
Feature/tweet link preview

### PR DESCRIPTION
**Pull request by Mykyta-Pysarev**
Added tweet link preview.
Screenshot with working element will be after backend [fix](https://github.com/attech-org/leleka-backend/pull/64)

### CODE REVIEW

**Review by DavidKorystov — APPROVED** [review]

File: src/components/FeedSingleTweet.tsx [hunk 1]
```diff
@@ -90,6 +91,12 @@ const FeedSingleTweet = ({
   // const handleReplyClick = () => {
   //   return <ReplyTweet />;
   // };
+  const urlSearch = new RegExp(
+    "/https?://(?:www.|(?!www))[a-zA-Z0-9][a-zA-Z0-9-]+[a-zA-Z0-9].[^s]{2,}|https?://(?:www.|(?!www))[a-zA-Z0-9]+.[^s]{2,}/"
+  );
+
+  const url = content.match(urlSearch) || "";
```
Comment (alexandrtovmach) [inline src/components/FeedSingleTweet.tsx]:
```suggestion
  const url = content.match(urlSearch) || [];
```
to keep consistency with types

**Review by alexandrtovmach — CHANGES_REQUESTED** [review]

**Review by VitaliiDenysiuk — CHANGES_REQUESTED** [review]
something was wrong
made tweet which has only url 
https://www.youtube.com/
leleka shows nothing - blank page at all

**Comment by Mykyta-Pysarev** [comment]
> something was wrong made tweet which has only url https://www.youtube.com/ leleka shows nothing - blank page at all

html tags also fetched as part of the link, when parsing. Now should be ok.

**Review by DavidKorystov — APPROVED** [review]

**Comment by Mykyta-Pysarev** [comment]
Fix some issues with fetch response other than "200":
image

**Pull request merged**